src/pages/Home.tsx

```diff
@@ -30,21 +66,54 @@ function Home() {
   }
 
   const cards = data?.businesses.map((element) => {
-    return <RestaurantPreview key={"home-" + element.alias} {...element} />;
+    const prevPropsComponent: PreviewPropComponent = {
+      vars: { ...element },
+      setId: setId,
+    };
+    return (
+      <RestaurantPreview
+        key={"home-" + element.alias}
+        {...prevPropsComponent}
+      />
+    );
   });
   return (
     <ContentWithSidebar
       sidebarPosition="left"
       sidebarWidth="1/5"
       sidebarBackground="backgroundOverlay"
     >
-      <Box
-        height="full"
-        display="flex"
-        justifyContent="center"
-        alignItems="center"
-      >
-        <Body size="large">filters</Body>
+      <Box height="full" padding={24}>
+        <Stack space={16} align="left">
+          {/* price filters */}
+
```

Comment:
The component for the price filter is defined, along with the API filter support, but here it's not used. If you decided to not include it in the app, it's better to remove the rest of the code to keep the codebase more compact and clean.